client/gefyra/cluster/resources.py

```diff
@@ -159,6 +160,33 @@ def create_operator_deployment(
     return deployment
 
 
+def _check_pod_for_command(pod: V1Pod, index=0):
+    containers: list[V1Container] = pod.spec.containers
+    if not len(containers):
+        raise RuntimeError(f"No container available in pod {pod.metadata.name}.")
+
+    ALLOWED_COMMANDS = ["sh", "bash", "zsh", "ash", "/entrypoint.sh"]
+
+    if (
+        containers[index].command
```

Comment:
This checks the command of the first defined container in the Pod which is not necessarily the container I want to bridge. The index is not passed when calling this function below.